src/implementations/vanilla/components/tabs/tabs.js

```diff
@@ -190,7 +190,7 @@ export class Tabs {
       newOffset = maxScroll;
     }
     this.list.style.transitionDuration = '0.4s';
-    this.list.style.transform = `translate3d(-${newOffset}px, 0px, 0px)`;
+    this.list.style.transform = `translate3d(${newOffset}px, 0px, 0px)`;
```

Comment:
this breaks the navigation for the non-rtl tabs in mobile, the pagination is in reverse, I won't apply this change now?

Reply:
yes, I did it for testing and forgot to remove... I'll update